src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -174,6 +174,8 @@ public partial class CreatureStageHUDBase<TStage> : HUDWithPausing, ICreatureSta
 
     // This block of controls is split from the reset as some controls are protected and these are private
 #pragma warning disable CA2213
+    [Export]
+    private Panel damageScreenEffect = null!;
```

Comment:
Can't this be just a base `Control` to give HUDs more flexibility (in case it is needed in the future) for some HUDs.